src/common/AllExperiments/Experiments_Live.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// The dependencies required to build this project are only available in the official build pipeline and are internal to Microsoft.
+// However, this project is not required to build a test version of the application.
+namespace AllExperiments
+{
+    using System.Text.Json;
+    using Microsoft.PowerToys.Settings.UI.Library.Telemetry.Events;
+    using Microsoft.PowerToys.Telemetry;
+    using Microsoft.VariantAssignment.Client;
+    using Microsoft.VariantAssignment.Contract;
+    using Wox.Plugin.Logger;
+
+#pragma warning disable SA1649 // File name should match first type name
+    public class Experiments
+#pragma warning restore SA1649 // File name should match first type name
+    {
+        public bool EnableLandingPageExperiment()
+        {
+            Experiments varServ = new Experiments();
+            varServ.VariantAssignmentProvider_Initialize();
+            var landingPageExperiment = varServ.IsExperiment;
+
+            return landingPageExperiment;
+        }
+
+        private void VariantAssignmentProvider_Initialize()
+        {
+            IsExperiment = false;
+
+            var vaSettings = new VariantAssignmentClientSettings
+            {
+                Endpoint = new Uri("https://default.exp-tas.com/exptas77/a7a397e7-6fbe-4f21-a4e9-3f542e4b000e-exppowertoys/api/v1/tas"),
+                EnableCaching = true,
+                ResponseCacheTime = TimeSpan.FromMinutes(5),
+            };
+
+            var vaClient = vaSettings.GetTreatmentAssignmentServiceClient();
+            var vaRequest = GetVariantAssignmentRequest();
+
+            try
+            {
+                var task = vaClient.GetVariantAssignmentsAsync(vaRequest);
```

Comment:
This is a network request in the initialization of this class. This means the UI will block waiting for this, right? Is there a way to make it so this can work asynchronously?